netjsonconfig/backends/zerotier/zerotier.py

```diff
@@ -20,12 +20,18 @@ def auto_client(
         name='ow_zt',
         nwid=None,
         identity_secret='{{zt_identity_secret}}',
+        config_path='',
+        copy_config_path='0',
+        zt_ifname='',
         disabled=False,
     ):
         nwid = nwid or ['']
         return {
             'id': nwid,
             'name': name,
             'secret': identity_secret,
+            'config_path': config_path,
+            'copy_config_path': copy_config_path,
```

Comment:
Don't let user to define this options

Reply:
If `ifname` is present then use default values for these options otherwise overwrite those params